core/templates/pages/exploration-editor-page/modal-templates/exploration-modify-translations-modal.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="modal-header">
+  <h4 class="modal-title">Modify Existing Translations</h4>
+</div>
+<div class="modal-body">
+  <p>
+    *Please update any translations below which are easy to update, and tick the ones you're
+    confident about. Those translations will be saved and shown directly to learners as soon
+    as you save the pending changes to the exploration.
```

Comment:
Should we use i18n for text here and below?

Reply:
I don't believe so since it's not a learner-facing string, such strings don't use i18n currently in the codebase from what I've seen